### PR TITLE
Map

### PR DESCRIPTION
**Pull request by zidder**
Armenia

### CODE REVIEW

File: bin/setup.sh [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 sudo apt-get install python3.7
 sudo apt-get install python3-tk
+sudo apt-get install libgeos-3.7
```
Comment (salbert96) [inline bin/setup.sh]: There was a problem when installing this package. So I installed a different version in docker. Is there a difference?
See the comment here.
https://stackoverflow.com/questions/38436607/whats-the-libgeos-3-4-2-so-and-how-to-install-this#comment64280488_38436607

File: visualization/globe.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import numpy as np
```
Comment (salbert96) [inline visualization/globe.py]: Could you please add user manual inside the README of the repo?
Even if it's just `python globe.py`

File: visualization/globe.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import matplotlib.pyplot as plt
+from mpl_toolkits.basemap import Basemap
+from matplotlib.collections import LineCollection
+from matplotlib import cm
+import shapefile
+
+from armenia import Province, province_data
+
+
+r = shapefile.Reader("armenia/gadm36_ARM_shp/gadm36_ARM_1")
```
Comment (salbert96) [inline visualization/globe.py]: Can we use absolute path here? By using the dirname of the file

Comment (salbert96) [reply visualization/globe.py]: PS: I already changed it inside my last commit.

File: bin/setup.sh [hunk 4]
```diff
@@ -2,3 +2,5 @@
...
+sudo apt-get install libgeos-dev
```
[the remark this answers is not in the document]
Comment (salbert96) [reply bin/setup.sh]: Actually, let's not focus on it right now. Since we have the docker workaround. We can worry about this later in other PRs.

**Comment by zidder** [comment]
Armenia

File: .gitignore [hunk 5]
```diff
@@ -9,4 +9,4 @@ __pycache__/
 
 .meteor
 
-node_modules
\ No newline at end of file
+node_modules
```
[the remark this answers is not in the document]
Comment (salbert96) [reply .gitignore]: sorry all good

**Comment by salbert96** [comment]
@zidder does this show the population size depending on the marker size? Images are great but including some text description too is super helpful in general. It does not require guessing 🙂

**Pull request merged**